spec/automation/unit/method_validation/amazon_pre_retirement_spec.rb

```diff
@@ -26,4 +26,10 @@
     MiqAeEngine.instantiate("#{@ins}?Vm::vm=#{@vm.id}#amazon", @user)
     expect(MiqQueue.exists?(:method_name => 'stop', :instance_id => @vm.id, :role => 'ems_operations')).to be_falsey
   end
+
+  it "calls abort for nil instances" do
+    @vm.hardware = nil
+    expect { MiqAeEngine.instantiate("#{@ins}?Vm::vm=#{@vm.id}#amazon", @user) }.to raise_error(
+      MiqAeException::AbortInstantiation)
+  end
```

Comment:
@billfitzgerald0120 
On line 5 you have
@ems  = FactoryGirl.create(:ems_vmware, :zone => @zone)

can we change this to
@ems  = FactoryGirl.create(:ems_amazon, :zone => @zone)